README.md

```diff
@@ -722,6 +797,52 @@ When we call `UserFactory::new()->withClients()()`, Poser understands that you'r
 UserFactory::new()->withClients(CustomerFactory::times(10))();
 ```
 
+#### Poser is creating more models than expected when handling relationships
+This happens when you declare default closures in your Laravel Factories. Poser has no way to detect that these exist, 
+so they are still created when Poser makes the models. Poser provides a suitable workaround to this in Poser defaults.
+If this issue affects you, we recommend stripping out the closures in your Laravel Factories and replacing them with 
+defaults in the relevant Poser Factory. Let's look at an example.
+
+```php
+$factory->define(Customer::class, function (Faker $faker) {
+    return [
+        'name' => $faker->name,
+        'user_id' => function() {
+            return factory(User::class)->create()->id;
+        }
+    ];
+});
+```  
+
+This `Customer` Laravel Factory has a closure for `user_id`, which causes n+1 `User` models to
+be created when calling `UserFactory::new()->withCustomers(n)->create()`. To resolve this issue,
+alter the above factory like so:
+
+```php
+$factory->define(Customer::class, function (Faker $faker) {
+    return [
+        'name' => $faker->name
+    ];
+});
+```  
+
+Now, we can alter the Customer Poser factory, like so:
+
+```php
+class CustomerFactory extends Factory {
+
+    public function defaultForUser()
+    {
+        return UserFactory::new()->withAttributes(['name' => "John Doe"]);
```

Comment:
Maybe just `return UserFactory::new()` since the callback this replaces doesn't specify a name attribute?

Reply:
Good idea, I agree. Pulled across from testing and by this point, my mind was a mess 😁👌

Sorted in latest push.